test/Twig/Tests/FileExtensionEscapingStrategyTest.php

```diff
@@ -28,12 +28,13 @@ public function getGuessData()
             array('html', 'foo'),
             array('html', 'foo.bar.twig'),
             array('html', 'foo.txt/foo'),
-            array('html', 'foo.txt/foo.js/'),
```

Comment:
i fixed it because otherwise it would not be consistent that a possible twig suffix is not cut for directories